naming_conventions.md

```diff
@@ -4,36 +4,36 @@ This document suggests some standards for the treatment of EUREC4A metadata.  Th
-  * **product_id:**  An optional identifier to be used with the instrument_id to allow groups the flexibility to group measurements in different ways, i.e., SAFIRE_Aerosol might designate a bundle of aerosol instruments run by SAFIRE.
-  * **variable_id:** A specific quantity. Usually this will be the output from a sensor, but it could also be a derived or composite variable.  Examples might be a voltage on a power supply, a temperature reading, a radar reflectivity, the latitude and longitude of a track, or maybe the divergence from a sounding circle. It should follow a standard vocabulary whenever possible, i.e., NetCDF Climate and Forecast (CF) Metadata Conventions.
-  * **time_id:** For some data it may be useful to indicate the time of the measurement or the time-range of the data included in the file. This will adopt the form yyyymmddTHHMMSS-yyyymmddTHHMMSS, or e.g., yyyymmdd for a single instance.
- * **version_id:** The data versioning, in the form vN.M with N and M integers. Using version 0 for preliminary data is encouraged.
+  * **campaign:**  EUREC4A
+  * **project:**  An optional identifier to allow groups to give additional specificity to a set of measurements collected with different platforms, for instance ATOMIC.
+  * **platform:**  An identifier drawn from a controlled vocabulary specifying a platform from which the data was collected.  See [EUREC4A.yaml](EUREC4A.yaml) which maps `platform` to platform metadata.
+  * **instrument:**  Specifies an instrument from which a measurement has been made.  Ideally similar instruments from different platforms will adopt similar identifiers. Level 2 data will often have a `platform` and `instrument`, but in some cases an instrument and platform may be synoymous.  
+  * **product:**  An optional identifier to be used with the `instrument` to allow groups the flexibility to group measurements in different ways, i.e., SAFIRE_Aerosol might designate a bundle of aerosol instruments run by SAFIRE.
+  * **variable:** A specific quantity. Usually this will be the output from a sensor, but it could also be a derived or composite variable.  Examples might be a voltage on a power supply, a temperature reading, a radar reflectivity, the latitude and longitude of a track, or maybe the divergence from a sounding circle. It should follow a standard vocabulary whenever possible, i.e., NetCDF Climate and Forecast (CF) Metadata Conventions.
+  * **time:** For some data it may be useful to indicate the time of the measurement or the time-range of the data included in the file. This will adopt the form yyyymmddTHHMMSS-yyyymmddTHHMMSS, or e.g., yyyymmdd for a single instance. `time` should be specified at the minimum level of complexity required to distinguish one file from another and should be specified in UTC.
+ * **version:** The data versioning, in the form vN.M with N and M integers. Using version 0 for preliminary data is encouraged.
```

Comment:
Following the discussion in #21, we should maybe add that the reason to include a version is that no two different files with the same name or maybe  "way of referencing" should exist. So it is ok to omit the version if another, clearly communicated way of referencing a specific revision of the dataset exists (e.g. git, DOI, etc...). And also if a version is part of the file name, the version **must always** be changed if the dataset content is changed.

Reply:
@d70-t Language around time specification has been clarified with examples, and to say that a single time represents the starting window. 

For the moment let's leave the version number as part of the file name, in which case it's clear that the number should be changed if the data are updated. 

If you agree you could merge.